Src/Base/AMReX_BLBackTrace.cpp

```diff
@@ -188,9 +189,29 @@ BLBackTrace::print_backtrace_info (FILE* f)
                 // cmd = "/usr/bin/eu-addr2line -C -f -i --pretty-print -p "
                 cmd = "/usr/bin/eu-addr2line -C -f -i -p "
                     + std::to_string(pid);
+
+                for (int i = 0; i < nentries; ++i)
+                {
+
+#if !defined(AMREX_USE_OMP) || !defined(__INTEL_COMPILER)
+                  std::string addr2line_result;
+                  if (amrex::system::call_addr2line && have_eu_addr2line) {
+                    if (bt_buffer[i] != nullptr) {
+                      char print_buff[32];
+                      std::snprintf(print_buff,sizeof(print_buff),"%p",bt_buffer[i]);
+                      const std::string full_cmd = cmd + " " + print_buff;
+                      addr2line_result = run_command(full_cmd);
+                    }
+                  }
+                  if (addr2line_result.find("AMReX_BLBackTrace.cpp:") != std::string::npos) {
+                    use_eu_addr2line = 1;
+                    break;
```

Comment:
This change would remove the break, and fix the equality:
```suggestion
                  if (addr2line_result.find("AMReX_BLBackTrace.cpp:") == std::string::npos) {
                    use_eu_addr2line = 1;
```

Reply:
I think it should be something like this
```
if (use_eu_addr2line == 0) { // still the initial value
    if (addr2line_result.find("AMReX_BLBackTrace.cpp:") != std::string::npos)) { // found AMReX_BLBackTrace.cpp:
        use_eu_addr2line = 1;
    }
}
```
Once `use_eu_addr2line` is 1, we don't have to do the test anymore.